CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 0.2.2
```

Comment:
```suggestion
## 0.3.0
```
I'd suggest to follow the semantic versioning https://semver.org/ which recommends the following:

```
Summary

Given a version number MAJOR.MINOR.PATCH, increment the:

    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards compatible manner, and
    PATCH version when you make backwards compatible bug fixes.

Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
```

Reply:
OK, based on this, I'll need to bump the major version.